examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -37,6 +37,9 @@ CHIP_ERROR PairingCommand::Run(PersistentStorage & storage, NodeId localId, Node
     params.mDeviceAddressUpdateDelegate = this;
     params.pairingDelegate              = this;
 
+    err = mCommissioner.SetUdpListenPort(storage.GetListenPort());
```

Comment:
I wonder if the UDP listen port should instead be part of the 'Init'.

Do we expect the UDP port to be changed/set up at a different point than init?

Reply:
@vivien-apple - should we create some PR later that puts this in the initializing parameters?